autodispose/src/main/java/com/uber/autodispose/ParallelFlowableScoper.java

```diff
@@ -0,0 +1,50 @@
+package com.uber.autodispose;
+
+import org.reactivestreams.Subscriber;
+
+import io.reactivex.Maybe;
+import io.reactivex.functions.Function;
+import io.reactivex.parallel.ParallelFlowable;
+
+class ParallelFlowableScoper<T> extends Scoper
+    implements Function<ParallelFlowable<? extends T>, ParallelFlowableSubscribeProxy<T>> {
```

Comment:
Since this is never going to be used with `to()`, let's make this implement the `ParallelFlowableConverter` directly and skip the indirection of the others. I mostly haven't migrated the others to this pattern yet to give users time to migrate, but they'll all be moved to this for 1.0. This one can start with it since it's new

Reply:
Alright, I'll make the change. You want me to open an issue for this and maybe we can work on it as a part of 0.7.0 release?

Reply:
Don't worry about it for now. I'll migrate the rest en masse for the 1.0 release when the time comes